applications/OptimizationApplication/python_scripts/execution_policies/stepping_analysis_execution_policy.py

```diff
@@ -1,42 +1,33 @@
-# ==============================================================================
-#  KratosOptimizationApplication
-#
-#  License:         BSD License
-#                   license: OptimizationApplication/license.txt
-#
-#  Main authors:    Suneth Warnakulasuriya
-#
-# ==============================================================================
-
 import KratosMultiphysics as Kratos
```

Comment:
I suggest stepping_analysis_wrapper class name

Reply:
addressed in general answer